stan/math/prim/scal/meta/return_type.hpp

```diff
@@ -9,15 +9,20 @@ namespace stan {
 /**
  * Metaprogram to calculate the base scalar return type resulting
  * from promoting all the scalar types of the template parameters.
+ * Note: Sub-double types are promoted to double.
```

Comment:
what is "sub-double"?

After rereading this a few times, is this what you're saying?
"Note: C++ primitive types are promoted to double."

Reply:
That’s probably what I’m saying. Anything that `promote_args` will promote to double will be. I didn’t check if that’s all the primitive types.